docs/rules/template-indent.md

```diff
@@ -0,0 +1,66 @@
+# ember/template-indent
+
+🔧 This rule is automatically fixable by the [`--fix` CLI option](https://eslint.org/docs/latest/user-guide/command-line-interface#--fix).
```

Comment:
how does this interact with prettier?

Reply:
when prettier is used, this rule should be disabled.

Reply:
can that be documented here?

should we also then add a `prettier` configuration that disables rules that prettier should handle?

Reply:
i do not think its required. This should remain disabled by default.

Reply:
Yeah, I believe we don't have any stylistic rules that conflict with prettier enabled by default/recommended.